openmetadata-service/src/main/java/org/openmetadata/service/apps/ApplicationHandler.java

```diff
@@ -130,6 +130,8 @@ public Object runAppInit(App app, CollectionDAO daoCollection, SearchRepository
   public void runMethodFromApplication(
       App app, CollectionDAO daoCollection, SearchRepository searchRepository, String methodName) {
     // Native Application
+    LOG.info("setting application runtime properties");
```

Comment:
do we need this log?

Reply:
at least on debug + app name 🤷

Reply:
no need for it.. it was a sanity check bc i wasnt sure I was deploying the right package